From 2b9bf99d69ad55c9db1ca8958640b6828431891b Mon Sep 17 00:00:00 2001
From: Siddharth Seth <sseth@apache.org>
Date: Mon, 18 Apr 2016 10:42:33 -0700
Subject: [PATCH 0890/1164] CDH-48076 : HIVE-13505 : Skip running TestDummy
 where possibe during precommit builds. (Siddharth
 Seth, reviewed by Ashutosh Chauhan)

Change-Id: I55a0bd63463d8b70b46c79d19875fe307581d296
---
 testutils/ptest2/src/main/resources/source-prep.vm |    2 --
 1 file changed, 2 deletions(-)

diff --git a/testutils/ptest2/src/main/resources/source-prep.vm b/testutils/ptest2/src/main/resources/source-prep.vm
index babc245..ca76f28 100644
--- a/testutils/ptest2/src/main/resources/source-prep.vm
+++ b/testutils/ptest2/src/main/resources/source-prep.vm
@@ -86,10 +86,8 @@ cd $workingDir/
   then
     rm -rf $workingDir/maven/org/apache/hive
     mvn -B clean install -DskipTests -Dmaven.repo.local=$workingDir/maven $mavenArgs $mavenBuildArgs
-    mvn -B test -Dmaven.repo.local=$workingDir/maven -Dtest=TestDummy $mavenArgs $mavenTestArgs
     cd itests
     mvn -B clean install -DskipTests -Dmaven.repo.local=$workingDir/maven $mavenArgs $mavenBuildArgs
-    mvn -B test -Dmaven.repo.local=$workingDir/maven -Dtest=TestDummy $mavenArgs $mavenTestArgs
   elif [[ "${buildTool}" == "ant" ]]
   then
     ant $antArgs -Divy.default.ivy.user.dir=$workingDir/ivy \
-- 
1.7.9.5

